xbmc/video/VideoInfoTag.cpp

```diff
@@ -592,7 +593,7 @@ void CVideoInfoTag::Serialize(CVariant& value) const
   value["playcount"] = GetPlayCount();
   value["lastplayed"] = m_lastPlayed.IsValid() ? m_lastPlayed.GetAsDBDateTime() : StringUtils::Empty;
   value["top250"] = m_iTop250;
-  value["year"] = m_premiered.GetYear();
+  value["year"] = GetYear();
```

Comment:
I suggest 
```
if (m_year > 0)
    value["year"] = m_year;
  else if (m_premiered.IsValid())
    value["year"] = m_premiered.GetYear();
  else
    value["year"] = 0;
```

Reply:
This is GetYear() and this does the almost the same as your suggestion:
```
int CVideoInfoTag::GetYear() const
{
  if (m_year > 0)
    return m_year;
  if (m_firstAired.IsValid())
    return GetFirstAired().GetYear();
  if (m_premiered.IsValid())
    return GetPremiered().GetYear();
  return 0;
}
```

Reply:
Ha, cross posted!
I didn't suggest GetYear because that potentially involves first aired date, and previous JSON results have been only from premiered date. However looking closer only episodes have a valid first aired date, and only movies and TV shows return "year" and "premiered" properties. So GetYear will work.

Note this also solves a previous issue with invalid premiered dates returning odd "year" values because validity was not checked.